.github/workflows/dependency-submission.yml

```diff
@@ -0,0 +1,25 @@
+# https://docs.github.com/en/code-security/dependabot/ecosystems-supported-by-dependabot/supported-ecosystems-and-repositories#gradle
+name: Dependency Submission
+
+run-name: Running Dependency Submission
+
+on:
+    workflow_dispatch:
```

Comment:
what process will call this workflow?

Reply:
Note that the reference here https://github.com/marketplace/actions/build-with-gradle#the-dependency-submission-action

And also the PostNl one get triggered by pushes to branches